lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -365,6 +367,18 @@ def add_address(post, options)
         end
       end
 
+      def add_statement_address(post, options)
+        return unless statement_address = options[:statement_address]
+        return unless [:address1, :city, :zip, :state].all? { |key| statement_address[key].present? } 
+
+        post[:statement_address] = {}
+        post[:statement_address][:line1] = statement_address[:address1]
+        post[:statement_address][:line2] = statement_address[:address2]
```

Comment:
You aren't checking for presence a address2 up above. Would setting this to nil affect the serialization at all?